addon/components/base/bs-form.js

```diff
@@ -431,6 +434,10 @@ export default Component.extend({
     resetSubmissionState() {
       this.set('isSubmitted', false);
       this.set('isRejected', false);
-    }
+    },
+
+    async submit() {
```

Comment:
this is duplicated

Reply:
I'm not sure what you mean. This is the `submit` action that's yielded by the component. We also implement the `submit` hook executed on submit events. Both are using the `submitHandler()`.

Reply:
Nevermind! In the diff view this looked as if was overriding the other `submit`. The `actions: {` part was not shown, so I got confused 🤦‍♂️😂